### DISCUSSION TITLE
Sentar Galaxy Main

### DISCUSSION THREAD

**Discussion by OcelotWalrus** [OcelotWalrus/Cromha-Expansion-plugin#112] [General]
It's a new galaxy where there will be a few Far Remnants system but mostly Uninhabited or Sheragi controlled systems. Most important will be a new species that has access to mysterious and old technology: The Sheragi. The Sheragi also keep contact with the Far Remnants and are very friendly.

**Some Lore:**
In the vanilla lore, it is said that the Quargs are the oldest living species after the Drak in the Milky Way but it is said that they not obligatory come from the Milky Way. Here is the lore:

The Quarg were at first a 'middle-aged' species in the Sentar Galaxy, that was developing and finding extremely powerful and mysterious technologies. It's when an invasive species was born: The Sheragis: They are a dragon-like species that have similarities to the Quarg: being reptilians and having similar technologies. It will not be exact in the lore, but a war would happen between them and the Quarg had to send eggs of their children and technologies to the Milky Way.

**Map:**
![image](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/bb824c7e-54d4-4617-a5c8-eaf8c40e7e59)

**How to get there:**
You will have to travel to the Helk'alk system, it's in the east of the Empire-Korath Space. Make sure you have a Mineria Key Stone.

**To-Do:**
- [x] Create the new species governments
- [x] Create lore
- [ ] Expand the Far Remnants
- [x] Create new outfits & ships models
- [ ] Create the small storyline that require to be done to access the system
- [ ] Implement Quarg-Sheragi war lore in the map and the storyline (add systems where you can find broken ringworld and other stuff like that)
- [ ] Edit `sheragi.txt`
- [ ] Edit `sheragi fleets.txt`
- [ ] Edit `sheragi missions.txt`
- [ ] Edit `sheragi modifications.txt`
- [ ] Edit `sheragi outfits.txt`
- [ ] Edit `sheragi ships.txt`
- [ ] Edit `sheragi variants.txt`
- [ ] Create additional outfits
- [ ] Create additional ships
- [ ] Create a small Sheragi storyline for the player get the two licenses, discover some lore and get offered a ship: the Emerald Sword.
- [ ] More

**Comment by lumbar527** [comment]
What if the new species used special drives with side effects?

**Reply by OcelotWalrus** [reply]
It is what I planned.
I already created the Warp Drive (a compact and lighter jump drive):
![warp drive](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/390abc5c-60a9-468b-9211-30f93a202cc1)
The Warp Drive is also used by the Far Remnants.

**Comment by OcelotWalrus** [comment]
I know what species it will be: the Sheragi!

**Reply by lumbar527** [reply]
Ooh! Note that I think there is a plugin that adds Sheragi systems, and that they use engines mixed with their drives.

**Reply by OcelotWalrus** [reply]
I know. I'm contributed on a plugin called "Sheragi-Rebirth". I will use the plugin code as a base and I'll revamp everything. But first I have to add ton of systems in the Sentar galaxy.

**Comment by OcelotWalrus** [comment]
I already have an idea on how to player will discover the new galaxy & the Sheragi:

At first, the Elk'alk system where there is the wormhole to access to the galaxy will be invisible.
If the player goes to the Sayaiban system (in the east of the Korath space), there is a 15% chance that there is a conversation when the player find a Sheragi Warfare system and take it in its cargo.

If the player go see any Quarg planet/station, some Quarg will ask the player on how he got this outfit. There will a conversation and at the end if the player has gone enough in the Cromha Empire storyline, the quarg will trust the player and tell him to go see the Far Remnants.

If the player didn't go far enough in the storyline, they will refuse to tell him.

When the player will go see the Far Remnants, they will say that it comes from a dragon-like species. The Far Remnants will give the player an outfit that lets him see the Helk'alk system so he can go to the new galaxy if he gave the correct answers to the Far Remnants.

**Reply by lumbar527** [reply]
That sounds cool!

**Reply by lumbar527** [reply]
Wait—what about if they did the (not yet) Lumbarian or Empire Rebel storylines?
The Quarg probably aren't happy about the Lumbarian Empire, due to the outfit duplication.

**Reply by OcelotWalrus** [reply]
Yes butt The Quarg can't know if they are friendly with the Lumbarian or the Empire Rebels. But in vanilla if you enter any Quarg planet/station with one of their outfits, you game over. So I don't think that's a problem.

Actually just so know it's because the Lumbarian Empire stolen Quarg technologies that the Cromha Empire are hostile the the Lumbarian Empire because the Qaurgs are part of the [Great Alliance](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/wiki/Governments-&-Major-Factions#the-great-alliance)

**Reply by lumbar527** [reply]
The Lumbarian Empire didn't exactly _steal_, they just scanned and replicated it. But it comes to the same thing.

> Yes but The Quarg can't know if they are friendly with the Lumbarian or the Empire Rebels.

How then would they know you are friendly with the Cromha Empire?

**Reply by OcelotWalrus** [reply]
> How then would they know you are friendly with the Cromha Empire?

If the player made till the part when he win the Empire Quantum Medal for going enough far in the storyline. So it will just check if the player has a Empire Quantum Medal and has a reputation higher that negative.

> The Lumbarian Empire didn't exactly steal, they just scanned and replicated it. But it comes to the same thing.

How could you possibly scan a ship and replicate it? It does not sound logical. Because for me I think the Quarg ships are made with special materials.

In the storyline I think what happened is that the Lumbarian captured a single Quarg Wardragon and was able to figure out what materials it was using and they were able to reproduce & synthesize a material that was looking the most possible with the Quarg ships material.

**Reply by OcelotWalrus** [reply]
I'm adding that to the wiki page for the Historical Timeline.

![image](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/df36b4fb-b208-42d2-b3e9-33932718cece)

**Reply by lumbar527** [reply]
OK, I guess that makes sense.

**Comment by OcelotWalrus** [comment]
You can now test the Sheragi outfits & ships by going to one of their systems.

**Comment by OcelotWalrus** [comment]
I've got a fucking good idea for the Sentar Galaxy lore:

As you may know, the Quargs are the oldest species after the Drak to be still alive in the Milky Way galaxy. But it is said that they are not obligatorily from the Milky Way. So what if Quargs came from the Sentar Galaxy but got invaded by the Sheragis, a species that just was just born when the Quarg moved from the Sentar Galaxy to the Milky Way?

**Reply by OcelotWalrus** [reply]
That would explain why Sheragi and Quarg have kind of similar tech and why they are both reptilian species! And also why you would have to go see the Quargs to show them the Sheragi debris you found in the storyline so they reveal to you the existence of the Sentar Galaxy.

**Reply by lumbar527** [reply]
I thought that the Quarg were from the milky way (https://github.com/endless-sky/endless-sky/pull/8721#issuecomment-1532857146). But still a cool idea!

**Reply by OcelotWalrus** [reply]
There is no real proof that they come from the Milky Way.
![image](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/b96e0593-402f-4b51-a773-b0563c6ffb70)
Taken from https://endless-sky.fandom.com/wiki/Quarg.

**Reply by OcelotWalrus** [reply]
Maybe instead of saying the Quarg had to leave the galaxy, we could say they send eggs of Quarg in the this Gas giant in the Milky Way in the hope that their children will develop enough. They also must have send technologies with them too. It's why the Quarg would say they come from the Milky Way.

**Reply by OcelotWalrus** [reply]
I'm editing all Quarg ships so instead they uses Jump Drives they will use Warp Drives.

**Reply by lumbar527** [reply]
Want to use my sprite from endless-sky/endless-sky#8721 for the Warp Drive, instead of a re-coloration?

**Reply by OcelotWalrus** [reply]
No thanks, check this, it is a pretty cool sprite I made:
https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/a471566c-97c8-4f4d-ae32-0cfa6ceb0839

Sorry my video editor bugged at the start.

**Reply by lumbar527** [reply]
I was thinking of the outfit image: https://github.com/lumbar527/endless-sky/raw/e13bbf448853fe530c0af79e6fbf481f313968f5/images/outfit/jump%20drive%20quarg.png

**Reply by OcelotWalrus** [reply]
Actually the Warp Drive already have a texture (not a recolor):
![warp drive](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/390abc5c-60a9-468b-9211-30f93a202cc1)

But if you want I could work on your model so it will be the Lumbarian Jump Drive.

**Reply by lumbar527** [reply]
Mostly I was thinking, because of comments about not just re-coloring the Jump Drive for my PR, I just though you might want to use a more unique/Quarg-like sprite.

If the Lumbarian Empire is getting their own jump drive, I'll make my own image.

**Reply by OcelotWalrus** [reply]
Okay, I actually made the Warp Drive myself in Blender, it's not just a recolor.

**Reply by lumbar527** [reply]
It's very similar though, but it's fine. Sorry if I seemed a little pushy or something.

**Reply by OcelotWalrus** [reply]
That's okay.

**Reply by OcelotWalrus** [reply]
Just so you see:
![warp drive](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/2530e07c-57d0-4ccd-867b-9d0e4a442681)
![jump drive](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/a356fc8a-4e71-40a1-8e4c-3822df901356)
 Actually the things that are around the Warp Drive are supposed to be snakes, because both Quarg and Sheragi are reptilians.

**Reply by lumbar527** [reply]
Cool! Speaking of drives, I'm working on one for the Lumbarian Empire. (I've already done the images and am just making the effects right now)

**Reply by OcelotWalrus** [reply]
I know I saw. By the way you set the range to 200 but that's way too much. I nerfed it to 160. So it is not overpowered but still really good. It may be the second best drive.

**Reply by OcelotWalrus** [reply]
If you want to keep talking about that, please do it in the Lumbarian Main discussion.

**Reply by lumbar527** [reply]
> There is no real proof that they come from the Milky Way.
> ![image](https://user-images.githubusercontent.com/87318892/255617456-b96e0593-402f-4b51-a773-b0563c6ffb70.png)
> Taken from https://endless-sky.fandom.com/wiki/Quarg.

> native to the Milk Way

I think that means from the Milky Way.

> although it is unknown where exactly the originate from **within the galaxy**.

Still inside the galaxy. Also, from the [Quarg Revamp Discussion](https://github.com/endless-sky/endless-sky/discussions/9016):
> Yes, that's what I mean - the Pug are Extragalactic, and the Quarg are not. It makes sense the Pug remain a mysterious factor.

From one of Saugia's replies.

**Reply by OcelotWalrus** [reply]
Yes there would be native from the Milky Way but their ancestors would be native from the Sentar Galaxy, are you following me?

**Reply by lumbar527** [reply]
Oooh! That works!

**Reply by OcelotWalrus** [reply]
I know :)
